src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/magical/MagicalZombiePills.java

```diff
@@ -61,4 +65,7 @@ public EntityInteractHandler getItemHandler() {
         };
     }
 
+    public ItemUseHandler onRightClick() {
+        return PlayerRightClickEvent::cancel;
+    }
```

Comment:
Mind adding a docs tag to that method to explain why it's cancelled?
For future reference in case someone stumbles across this and gets confused

Reply:
Like this?
```suggestion
    /**
     * This method cancels {@link PlayerRightClickEvent} to prevent placing Magical Zombie Pills.
     */
    public ItemUseHandler onRightClick() {
        return PlayerRightClickEvent::cancel;
    }
```

Reply:
Yeah, but you forgot the `@return` tag and you could make "Magical Zombie Pills" an `@link` too :)